tests/unit/utils/test_pkg.py

```diff
@@ -45,3 +50,62 @@ def test_split_comparison(self):
             oper, verstr = salt.utils.pkg.split_comparison(test_parameter[0])
             self.assertEqual(test_parameter[1], oper)
             self.assertEqual(test_parameter[2], verstr)
+
+
+@skipIf(NO_MOCK, NO_MOCK_REASON)
+@skipIf(pytest is None, 'PyTest is missing')
```

Comment:
I see nothing pytest-specific about these tests, so why is it being skipped?

Reply:
Well, this is a general pattern, apparently. Maybe we should stop even doing it, since pytest is always there.